test/train-sets/ref/cb_bad_one_of_string.stdout

```diff
@@ -1 +1 @@
-[critical] vw (options.h:345): Error: 'bad_type' is not a valid choice for option --cb_type. Please select from {dm, dr, ips, mtr, sm}
```

Comment:
Hopefully this change won't happen so much anymore :)

Reply:
Haha yes I hope so!